libcontainer/specconv/spec_linux.go

```diff
@@ -772,7 +772,8 @@ func CreateCgroupConfig(opts *CreateOpts, defaultDevs []*devices.Device) (*confi
 				c.Resources.CPUIdle = r.CPU.Idle
 			}
 			if r.Pids != nil {
-				c.Resources.PidsLimit = r.Pids.Limit
+				l := r.Pids.Limit
+				c.Resources.PidsLimit = &l
```

Comment:
I think maybe we can use `c.Resources.PidsLimit = &r.Pids.Limit` or `*c.Resources.PidsLimit = r.Pids.Limit`?

Reply:
I think `c.Resources.PidsLimit = &r.Pids.Limit` should be able to used in here.

Reply:
personally I'm anxious about bugs where the `r.Pids.Limit` is updated but `c.Resources.PidsLimit` isn't supposed to be updated. I would be surprised fi that happened but I feel like the code as-is future proofs against that